[Doc] Document Alluxio does not sync metadata from S3 by default [skip ci] (#7080)
NVIDIA/spark-rapids: docs/get-started/getting-started-alluxio.md

### Auto start Alluxio the master and workers
After installing Alluxio master and workers, it's better to add a systemd service for each process of master and workers.
Systemd service can automatically start process if process is terminated.

## Alluxio limitations
### Alluxio does not sync metadata from UFS(e.g. S3) by default

> Alluxio does not sync metadata from S3 by default, so it won't pick up changed files.

Reviewer: I'd also like to see a sentence at the very beginning like line 19 that says something like:  sets up Alluxio to handle reads and by default does not handle file updates in the remote blob store. 

We should also explicitly put somethign in here about writes don't go through alluxio.

Author: Done

### Auto start Alluxio the master and workers
After installing Alluxio master and workers, it's better to add a systemd service for each process of master and workers.
Systemd service can automatically start process if process is terminated.

## Alluxio limitations
### Alluxio does not sync metadata from UFS(e.g. S3) by default

> Alluxio does not sync metadata from S3 by default if writes don't go through alluxio, so it won't pick up changed files generated by external systems.

Reviewer: ```suggestion
Alluxio does not sync metadata from S3 by default so it won't pick up any changed files.
```

Author: Done

For example:
If you update a file in the S3 from 1M size to 10M size and Alluxio already cached the 1M size file, Alluxio cluster
will always use the 1M file.

> If you want to enable sync it has performance impact.

Reviewer: ```suggestion
If you want to enable sync it has performance impact which will affect the read performance.
```

Author: Done
